src/msg/async/dpdk/DPDK.cc

```diff
@@ -774,23 +789,9 @@ bool DPDKQueuePair::rx_gc(bool force)
   if (_num_rx_free_segs >= rx_gc_thresh || force) {
     ldout(cct, 10) << __func__ << " free segs " << _num_rx_free_segs
                    << " thresh " << rx_gc_thresh
-                   << " free pkts " << _rx_free_pkts.size()
+                   << " free pkts " << _rx_free_bufs.size()
                    << dendl;
 
-    while (!_rx_free_pkts.empty()) {
-      //
-      // Use back() + pop_back() semantics to avoid an extra
-      // _rx_free_pkts.clear() at the end of the function - clear() has a
-      // linear complexity.
-      //
-      auto m = _rx_free_pkts.back();
-      _rx_free_pkts.pop_back();
-
-      if (!refill_one_cluster(m)) {
-        ldout(cct, 1) << __func__ << " get new mbuf failed " << dendl;
-        break;
-      }
-    }
     for (auto&& m : _rx_free_bufs) {
```

Comment:
`refill_one_cluster` is commented out, so `_rx_free_bufs` is always empty, leads to rx_gc  doesn't work. Am I right？